test/tests/functional/pbs_qmgr.py

```diff
@@ -159,3 +208,143 @@ def test_multi_attributes(self):
             self.logger.info("looking for error, %s" % msg)
             self.assertTrue(msg in e.msg[0])
         self.server.expect(NODE, 'queue', op=UNSET, id=self.mom.shortname)
+
+    @tags('multisched')
+    def test_list_sched(self):
+        """
+        Test to verify that qmgr list sched works for multiple schedulers
+        """
+        # Scenario 1: only default sched present
+        # Scenario 2: multiple schedulers present
+
+        
+        # Scenario 1: only default sched present
+        self.server.manager(MGR_CMD_LIST, SCHED, id="default", expect=True,
+                                runas=ROOT_USER)
```

Comment:
Why are you running as ROOT_USER?  If you need to run as root, use sudo=True, but none of these commands require to run as root.  You can list sched as a normal user.